ioctl/newcmd/config/config.go

```diff
@@ -66,8 +66,9 @@ func InitConfig() (config.Config, string, error) {
 	// Load or reset config file
 	err = info.loadConfig()
 	if os.IsNotExist(err) {
-		err = info.reset() // Config file doesn't exist
-	} else if err != nil {
+		err = info.reset()
```

Comment:
could the reset info be directly returned?

Reply:
> could the reset info be directly returned?

if directly returned,  `err != nil` will handle twice.

Reply:
Sorry, I don't understand why it is handled twice?